bash-utils/queueligprep

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# script to submit Schrödinger commands using filename from "queue.txt"
+# only submits new jobs when current joblist is shorter than a threshold
+# execute this script frequently through cronjob to use it as queueing system
+# example crontab
+# * * * * * cd /path/to/log_out_dir && /path/to/queueligprep
+
+SCHRODINGER=/opt/schrodinger/suites2023-2
+WORKDIR=/Users/julian/SciAdv\ revisions
```

Comment:
nit: what about `WORKDIR=PATH_TO_YOUR_DIR` and add a comment saying `# remember to change WORKDIR to where your 'queue.txt' is in`

Reply:
changed defaults + added comments in new commit. The command line argument idea is nice, but I think overkill for this simple script.